doc/ide-setup.md

```diff
@@ -160,34 +161,55 @@ Following the above should give you the below result:
 
 For additional details on issues you may face when setting up REPL with Cursive [see this document](https://notes.status.im/9Gr7kqF8SzC_SmYK0eB7uQ?view#Connecting-Cursive--IntelliJ-IDEA-to-REPL-Problems)
 
-## Using Calva
+## Visual Studio Code
```

Comment:
FYI: Here's the rendered markdown from this branch: https://github.com/status-im/status-mobile/blob/vscode-configuration/doc/ide-setup.md